apps/website/versioned_docs/version-v2.x/use-cases/governance.md

```diff
@@ -1,8 +1,18 @@
 ---
-title: MACI Governance applications
+title: Governance Applications
 description: Overview of MACI governance applications.
 sidebar_label: Governance
 sidebar_position: 1
 ---
 
-MACI is a privacy-preserving, on-chain voting protocol. It is designed to be used in a variety of applications where stakes are high, and we believe that governance is one of them. At this time, no governance applications have been built on top of MACI, but we are excited to see what the community will build, and kick-start experiments in the following months. Stay tuned.
+MACI is designed to be used in a variety of applications where stakes are high, and we believe that governance is one of them.
+
+## DAOs Governance
+
+As a privacy-preserving, on-chain voting protocol, MACI can be used to run on-chain proposals were results execute smart contracts. Currently, public voting is problematic for DAOs as there is vote coercion, and bandwagon effects.
```

Comment:
I think the ***were*** should be ***where***?
and to make this sentence easier to understand, maybe could change to this:

MACI, as a privacy-preserving on-chain voting protocol, enables the execution of on-chain proposals where results directly trigger smart contracts. Public voting in DAOs currently faces challenges such as vote coercion and bandwagon effects, which MACI aims to mitigate.
(suggested by ChatGPT)